pcmanfm/desktopwindow.cpp

```diff
@@ -534,7 +534,10 @@ void DesktopWindow::paintBackground(QPaintEvent *event) {
     painter.fillRect(event->rect(), QBrush(bgColor_));
   }
   else {
-    painter.drawPixmap(event->rect(), wallpaperPixmap_, event->rect());
+    if(wallpaperMode_ == WallpaperTile)
```

Comment:
What do you mean by that? After @PCMan's workaround, tiling should be done by drawTiledPixmap().